test/tests/functional/pbs_offline_vnodes.py

```diff
@@ -489,8 +489,9 @@ def test_pbsnodes_multi_mom_offline_online(self):
         self.verify_vnodes_state('offline')
 
         # Now call pbsnodes -r to clear the offline from MomA
-        pbsnodes_clear_offline = [pbsnodes_cmd,'-r', momA.shortname]
-        self.du.run_cmd(self.server.hostname, cmd=pbsnodes_clear_offline, sudo=True)
+        pbsnodes_clear_offline = [pbsnodes_cmd, '-r', momA.shortname]
+        self.du.run_cmd(self.server.hostname, cmd=pbsnodes_clear_offline,
+                        sudo=True)

```

Comment:
Do you need to run this as sudo?  You are added as a manager by PTL.  That should be sufficient.